WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderCommentListActivity.java

```diff
@@ -148,6 +162,27 @@ public void onClick(View v) {
         }
     }
 
+
+    private void reloadPostAndComments() {
```

Comment:
Can we change the name of this method to `updatePostAndComments()`? In most of the reader code, the word `reload` (as well as `refresh`) is used when loading from the local cache, whereas `update` is used when requesting data from the backend.